src/components/booksComp.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook, removeBook } from '../redux/books/books';
+import store from '../redux/configureStore';
+
+const BooksComponent = () => {
+  const [trigger, fireTrigger] = useState(true);
+  const dispatch = useDispatch();
+  let bookInputValue = '';
+  let authorInputValue = '';
+  let catInputValue = '';
+
+  const handleBookInputWr = () => {
+    bookInputValue = document.getElementById('newBookInp').value;
+  };
+  const handleAuthorInputWr = () => {
+    authorInputValue = document.getElementById('newAuthInp').value;
+  };
+  const handleCatInputWr = () => {
+    catInputValue = document.getElementById('newCatInp').value;
+  };
+
+  const clearInput = () => {
+    document.getElementById('newBookInp').value = '';
+    document.getElementById('newAuthInp').value = '';
+    document.getElementById('newCatInp').value = '';
+  };
+
+  const reRender = (callback, state) => {
+    if (state) {
+      callback(false);
+    } else {
+      callback(true);
+    }
+  };
+
+  const submitBookToStore = (c, t, a) => {
+    dispatch(addBook(c, t, a));
+  };
+  const removeBookFromStore = (i) => {
+    dispatch(removeBook(i));
+  };
+
+  return (
+    <div>
+      {trigger}
+      <ul>
+        {store.getState().booksReducer.map((array) => (
```

Comment:
- [ ] OPTIONAL - You can simply create a variable called books and assign it to the value of books in the store. It would make the code more concise and readable.